test-suite/src/function/cast.rs

```diff
@@ -51,9 +54,10 @@ test_case!(cast_literal, async move {
             r#"SELECT CAST("foo" AS INTEGER) AS cast FROM Item"#,
             Err(ValueError::LiteralCastFromTextToIntegerFailed("foo".to_owned()).into()),
         ),
+
         (
             r#"SELECT CAST(1.1 AS INTEGER) AS cast FROM Item"#,
-            Ok(select!(cast I64; 1)),
+            Err(ValueError::LiteralCastToDataTypeFailed(DataType::Int, "1.1".to_string()).into()),
```

Comment:
Wow, nice, this is even much better than removing the codes which I mentioned.
Thanks!